packages/react/src/primitives/Link/Link.tsx

```diff
@@ -10,6 +10,7 @@ import {
 } from '../types';
 import { View } from '../View';
 import { primitiveWithForwardRef } from '../utils/primitiveWithForwardRef';
+import { BiLinkExternal } from 'react-icons/bi';
```

Comment:
We don't usually pull in icons from an external source, we add them manually in the ui package. You'll also need to add it to the IconsProvider so it can be overridden.

Reply:
+1 to the above. Additionally, `react-icons` is not a dependency of `@aws-amplify/ui-react` and shipping this would cause runtime failures

Reply:
Done